### PR TITLE
Fix issues related to new Pandas version

### PR DESCRIPTION
**Pull request by aloukina**
This PR addresses #534 , #533  , and #531 

* Fixed test set up in TestAnalyzer
* Changed read_jsonlines to accommodate new behavior of `pd.normalize_json`
* Added "axis" keyword to instances of `pandas.dataframe.drop`

### CODE REVIEW

**Comment by aloukina** [comment]
I kept the error catching in `read_jsonlines` for compatibility with earlier Pandas version: that line won't be covered by tests.

File: rsmtool/reader.py [hunk 1]
```diff
@@ -63,8 +63,18 @@ def read_jsonlines(filename, converters=None):
 
     dfs = []
     for column in df:
+        # let's try to normalize this column
         try:
             df_column = pd.json_normalize(df[column])
+
+            # from Pandas 1.3 we get an empty df
+            # if the column does not contain a nested json
+            # in this case we simply copy the column
```
Comment (desilinguist) [inline rsmtool/reader.py:72]:
```suggestion
            # Starting with Pandas v1.3, we get an empty data frame
            # if the column does not contain a nested json. 
            # If this is the case, we simply copy the column.
```

File: rsmtool/reader.py [hunk 2]
```diff
@@ -63,8 +63,18 @@ def read_jsonlines(filename, converters=None):
...
+            if df_column.empty:
+                df_column = df[column].copy()
+
+        # Pandas <1.3 will raise an attribute error:
+        # we'll catch that too
```
Comment (desilinguist) [inline rsmtool/reader.py:77]:
```suggestion
        # Pandas <v1.3 will raise an attribute error instead, 
        # so we'll catch that too
```

**Review by desilinguist — CHANGES_REQUESTED** [review]
Looks great! Just some minor tweaks to the code comments.

**Comment by aloukina** [comment]
Thank you, @desilinguist  - all fixed!

**Review by desilinguist — APPROVED** [review]

**Pull request merged**